src/catalog/specs.rs

```diff
@@ -11,6 +11,15 @@ use std::time::Duration;
 #[schemars(example = "CollectionName::example")]
 pub struct CollectionName(#[schemars(schema_with = "CollectionName::schema")] String);
 
+impl<T> From<T> for CollectionName
```

Comment:
I don't think this should be an auto-magic conversion. If you want a CollectionName, you have to explicitly ask for one.

Otherwise it defeats the purpose of the newtype pattern, if I can convert (say) a RelativeUrl to a CollectionName via , eg, `.as_ref().clone().into()`. I can't read that and immediately know that I've subverted the type system.